src/main/java/de/tum/in/www1/artemis/service/LectureUnitProcessingService.java

```diff
@@ -130,6 +132,10 @@ private boolean isBreakSlide(String slideText) {
         return slideText.contains("Break") || slideText.contains("Pause");
     }
 
+    private boolean isSolutionSlide(String slideText) {
+        return slideText.contains("Example solution") || slideText.contains("Example solution (with comments)") || slideText.contains("Example solution: model");
```

Comment:
If it contains `Example solution (with comments)` then `Example solution` is also included.
```suggestion
        return slideText.contains("Example solution");
```

Reply:
you are right 👍